src/helpers/helpers.ts

```diff
@@ -34,10 +36,10 @@ export const checkZero = (amount: string) => {
   }
 };
 
-export const agreedCode = (code: string) => {
-  //TODO: till we not connect backend hard code this code
-  const tempCode = '1111';
-  if (tempCode !== code) {
+export const agreedCode = (userCode: string, email: string, pin: string) => {
+  const tempDecryptedCode = decryptCode(email, pin);
+
+  if (tempDecryptedCode !== userCode) {
```

Comment:
return tempDecryptedCode === userCode would be simpler

Reply:
done